### DISCUSSION TITLE
将「度／百分比與數字之間不需要增加空格」推广至「所有以符号表示的单位」

### DISCUSSION THREAD

**Discussion by iBug** [sparanoid/chinese-copywriting-guidelines#157] [Ideas]
本仓库内容：

> 例外：度／百分比與數字之間不需要增加空格

因此以下用法是正确的：

> 5 GB、100% 保障、36.5°C

但是，尽管以下情况并不属于「度／百分比与数字之间」，它们也应当属于「不增加空格」的例外情况：

- 时间和角度（分和秒）：3'54"
- 表示序数（ordinal）的符号：2\# 楼、`#2` 楼
- 货币：￥10、£10、€10
  - 该规则仅适用于符号与数字相邻时，因此应当写作 10 CHF 而不是 10CHF
  - 争议情况：货币并非只有符号时，如 US$10 和 US$ 10 看似均可（个人倾向于无空格的写法）
- 其他「类似」符号的情况，如使用上标小写字母 C 作为「货币分」时：2<sup>c</sup> 而不是 2 <sup>c</sup>

基于以上讨论，我认为如标题所述是一个很好的改进。

**Comment by HumJ0218** [comment]
我个人认为，百分比、度分秒不加空格是因为在数学上是属于数字本身，例如 100% 就是 1、0°30' 就是 0.5
但 GB 属于一种单位量，不是数字本身，所以要加空格分开

**Reply by iBug** [reply]
用作数学上的意义时，`°`、`'`、`"` 这些可以被认为属于数字本身，但在其他语义中（特别是时间，也包括经纬度等）它们应当被视作单位。同时货币符号不太能“属于数字本身”吧……
